docker/docker-compose.yml

```diff
@@ -11,7 +11,8 @@ services:
     volumes:
       - dbvolume:/var/lib/mysql
   codedefenders:
-    image: "codedefenders/codedefenders:${CODEDEFENDERS_VERSION:?Please set a Codedefenders version via the .env file. Also see docs/Docker.md for more information.}"
+    #image: "codedefenders/codedefenders:${CODEDEFENDERS_VERSION:?Please set a Codedefenders version via the .env file. Also see docs/Docker.md for more information.}"
+    image: "codedefenders/codedefenders:dev"
```

Comment:
I think this is good as is, the instructions specify how to do it correctly

Reply:
Ah yes, forgot it is set in .env file :+1: